fastify.js

```diff
@@ -620,11 +620,20 @@ function fastify (options) {
       return
     }
 
-    const body = JSON.stringify({
-      error: http.STATUS_CODES['400'],
-      message: 'Client Error',
-      statusCode: 400
-    })
+    let body
+    if (err.code === 'ERR_HTTP_REQUEST_TIMEOUT') {
```

Comment:
cant we avoid the unecessary stringification?

```js
const body = err.code === 'ERR_HTTP_REQUEST_TIMEOUT'
  ? '{"error":"Request Timeout","message":"Client Timeout","statusCode":408}')
  : '{"error":"Bad Request","message":"Client Error","statusCode":400}'
```

Shouldnt in line 641 be modified? The line with `this.log.trace({ err }, 'client error')` <-- should it not be also in timeout case be 'client timeout'?  or so?

Reply:
I double checked and my code was actually wrong when writing the response. I updated with a more general approach.

Reply:
looks nicer